sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -410,6 +429,21 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(false)
 
+  // METRICS
+
+  val METRICS_LEVEL = conf("spark.rapids.sql.metrics.level")
+      .doc("GPU plans can produce a lot more metrics than CPU plans do. In very large " +
+          "queries this can sometimes result in going over the max result size limit for the " +
+          "driver. Supported values include DEBUG: which will enable all metrics supported and " +
+          "typically only needs to be enabled when debugging the plugin. MODERATE: which is the " +
+          "default value and should output enough metrics to understand how long each part of " +
```

Comment:
Nit: Don't think it's necessary to mention the default value since it appears both here and in the generated documentation next to this text.